### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by glcohen**
I removed a few Workday-specific references. Open to suggestions to move forward.

### CODE REVIEW

File: Verifiable Credentials JSON Schema Specification.md [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+    },
+    "required": ["emailAddress"],
+    "additionalProperties": false
+  },
+  "proof": {
+      "created": "2019-09-27T06:26:11Z",
+      "creator": "did:work:MDP8AsFhHzhwUvGNuYkX7T#key-1",
+      "nonce": "0efba23d-2987-4441-998e-23a9d9af79f0",
+      "signatureValue": "2A7ZF9f9TWMdtgn57Y6dP6RQGs52xg2QdjUESZUuf4J9BUnwwWFNL8vFshQAEQF6ZFBXjYLYNU4hzXNKc3R6y6re",
+      "type": "Ed25519VerificationKey2018"
```
Comment (OR13) [inline Verifiable Credentials JSON Schema Specification.md]: This signature still looks very close to JSON-LD Signature. I'd like to recommend we either use JSON-LD signatures with detached jws (such as this example): https://github.com/decentralized-identity/lds-ecdsa-secp256k1-2019.js#verify

Or we find a way to use another form of detached jws without JSON-LD, pre processing. But I think thats better to tackle in a subsequent PR.

Comment (glcohen) [reply Verifiable Credentials JSON Schema Specification.md]: Agree, and agree this should be in a subsequent PR

**Comment by OR13** [comment]
@kimdhamilton @msporny how long should approved PRs remain open for additional comments?

I'm in favor of merging this and addressing comments / issues in subsequent PRs.

File: Verifiable Credentials JSON Schema Specification.md (hunk 1 above)
Comment (OR13) [reply Verifiable Credentials JSON Schema Specification.md]: I have captured this discussion in the following issue: https://github.com/w3c-ccg/vc-json-schemas/issues/2

I'm marking this as resolved, and we should be ready to merge once we hear back on PR comment duration.

**Comment by msporny** [comment]
> @msporny how long should approved PRs remain open for additional comments?

Just my opinion as an editor - we don't want to slow things down with CCG work items, especially this early in the process. It should be at the discretion of the editors since they pay the price of merging things too early (cranky comments from CCG members). Most people don't pay too much attention to documents this early in the process, so it's an opportunity to rapidly iterate. So, if the editors think it's ready, merge away.

I defer to @kimdhamilton @jandrieu and @ChristopherA on the Chairs position.

**Comment by OR13** [comment]
roger that, prepare for the jump to hyperspace...

**Comment by msporny** [comment]
For controversial PRs in WGs that seem to have consensus to merge, giving a 7 day warning before merging (to capture any objections from people not involved in the discussion) is typical. 

That said, I hope we don't resort to that sort of thing with CCG work items except for potentially at the very end of the Final Community Group Report process.

**Pull request merged**